lib/basedriver/commands/find.js

```diff
@@ -111,7 +110,8 @@ helpers.findByImage = async function (b64Template, {checkingStaleness = false} =
 };
 
 helpers.ensureTemplateSize = async function (b64Template, screenWidth, screenHeight) {
-  let {width: tplWidth, height: tplHeight, imgObj} = await getJimpData(b64Template);
+  let imgObj = await imageUtils.getJimpData(b64Template);
```

Comment:
is the function called `getJimpImage` or `getJimpData` ?

Reply:
oops, fixed.